spring-cloud-stream-binders/pom.xml

```diff
@@ -25,48 +22,6 @@
 		<module>spring-cloud-stream-binder-kafka</module>
 	</modules>
 	
-	<dependencyManagement>
```

Comment:
As per my other comment, let's consolidate this in a single <depenndencyManagement> section.